charmcraft/commands/store/store.py

```diff
@@ -59,3 +60,15 @@ def whoami(self):
             userid=response['id'],
         )
         return result
+
+    def register_name(self, name):
+        """Register the specified name for the authenticated user."""
+        self._client.post('/v1/charm', {'name': name})
```

Comment:
I'm rather surprised that all you do is post a name to charm. It seems like it should be more than just that. But if this is the API, then it is what it is.

Reply:
Yeap, that is it.